Split languages from types, define migration plan

joshreisner opened the pull request:
this PR:

* syncs language codes with widely-used browser language codes
* moves languages out of types into their own "namespace" to address the fact that we will want to keep adding languages to the spec and these can clash with type definitions
* proposes a migration plan to minimize hardship on current users

L-squared commented:
I noticed that the ISO languages don't list Hebrew, but aa-israel.org is sending in meetings with the type Hebrew applied. Any ideas on how that will migrate? Maybe the ISO resource isn't correct?

joshreisner commented:
I do see Hebrew defined as `he` on this page: https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes

L-squared commented:
OK - my mistake - I was taking the list in the readme file literally
 
![image](https://user-images.githubusercontent.com/10970074/196507671-305a78a3-0281-487d-a748-6bd4947f49e4.png)

joshreisner commented:
oops good catch, thanks. i've added it to the readme. let me know if you see any others that we should be supporting. i took the list that we had (or tried to) and added the ones that OIAA has on their site. but we can add any others from the ISO-639-1 list that we think we will need

joshreisner commented:
out of date - if this is still needed we'll want to start a new branch
